src/DepartmentService.Data/DepartmentRepository.cs

```diff
@@ -69,9 +69,14 @@ public async Task<(List<DbDepartment> dbDepartments, int totalCount)> FindAsync(
     {
       IQueryable<DbDepartment> dbDepartments = _provider.Departments.AsQueryable();
 
-      if (!filter.IncludeDeactivated)
+      if (filter.IsActive.HasValue)
       {
-        dbDepartments = dbDepartments.Where(d => d.IsActive);
+        dbDepartments = dbDepartments.Where(d => d.IsActive == filter.IsActive);
+      }
+
+      if (filter.IsAscendingSort.HasValue)
+      {
+        dbDepartments = filter.IsAscendingSort == true ? dbDepartments.OrderBy(d => d.Name) : dbDepartments.OrderByDescending(d => d.Name);

```

Comment:
move ? to the 80th line, : to the 81st line

Reply:
filter.IsActive.Value ? ... : ...

Reply:
done